.pre-commit-config.yaml

```diff
@@ -1,8 +1,4 @@
 repos:
-  - repo: https://gitlab.com/pycqa/flake8
```

Comment:
Please roll back these changes, or explain why they should be upstreamed.

Reply:
@g-nikoloff tells me he is having trouble with `pre-commit.com`'s hooks accessing https://gitlab.com/pycqa/flake8. 

Is this hook correct? Did this move to [github.com/PyCQA/flake8](https://github.com/PyCQA/flake8)?

Reply:
I have created https://github.com/google/capirca/pull/339 to fix this hook.

I suggest rebasing on top of my commit.

Reply:
I also changed it in my commit, so there wont be any conflict and we can merge it without any problems.